src/runtime/local/datastructures/CMakeLists.txt

```diff
@@ -29,4 +29,6 @@ target_link_libraries(DataStructures PUBLIC Proto)
 if(USE_CUDA AND CMAKE_CUDA_COMPILER)
     target_include_directories(DataStructures PUBLIC ${CUDAToolkit_INCLUDE_DIRS})
     target_link_libraries(DataStructures PUBLIC CUDA::cudart)
-endif()
\ No newline at end of file
+endif()
+target_compile_options(DataStructures PUBLIC "-mfma")
```

Comment:
This compile option troubles the Nvidia compiler. Compilation completed nevertheless :shrug: